content/classes-part-2/exercises/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Exercises: Objects and Classes Part 2"
+date: 2023-06-14T16:06:05-05:00
+draft: false
+weight: 3
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Work on these exercises in the IntelliJ `java-web-dev-projects` project. You will update your `Student.java` file by implementing the `addGrade` and `getGradeLevel` methods that were sketched out in the [Instance Methods]({{< relref "../reading/instance-and-static-methods/_index.md" >}}) section.
```

Comment:
Suggestion add link to project and update name to java-web-dev-projects-17 and provide path to where the starter code is.

Reply:
pathway added - we are using the `java-web-dev-projects` project instead of the `java-web-dev-projects-17`.